examples/viewer.html

```diff
@@ -380,6 +408,14 @@
     };
 
 
+    Viewer.prototype.sendRequest_ = function(type, data, awaitResponse) {
+      if (!this.messaging_) {
```

Comment:
Visibility is now processed separately. It's resent as soon as the comms are established. Since the viewer is the initiator of the protocol, it's not a good idea to buffer messages here - instead they should only be sent after handshake is done.